chia/wallet/wallet_protocol.py

```diff
@@ -19,16 +19,18 @@
 if TYPE_CHECKING:
     from chia.wallet.wallet_state_manager import WalletStateManager
 
+T = TypeVar("T", contravariant=True)
```

Comment:
This is where you can require that the types are `Streamable`.

```suggestion
T = TypeVar("T", bound=Streamable, contravariant=True)
```